app/src/main/java/com/orange/ouds/app/ui/components/radiobutton/RadioButtonDemoScreen.kt

```diff
@@ -0,0 +1,123 @@
+import com.orange.ouds.app.ui.components.enabledArgument
+import com.orange.ouds.app.ui.components.onClickArgument
+import com.orange.ouds.app.ui.utilities.composable.CodeSnippet
+import com.orange.ouds.app.ui.utilities.composable.CustomizationBottomSheetScaffold
+import com.orange.ouds.app.ui.utilities.composable.CustomizationSwitchListItem
+import com.orange.ouds.app.ui.utilities.composable.DemoScreen
+import com.orange.ouds.app.ui.utilities.composable.DetailScreenDescription
+import com.orange.ouds.app.ui.utilities.composable.LightDarkDemo
+import com.orange.ouds.core.component.OudsRadioButton
+import com.orange.ouds.core.theme.OudsTheme
+import com.orange.ouds.core.theme.OudsThemeTweak
+import com.orange.ouds.core.utilities.OudsPreview
+import com.orange.ouds.foundation.utilities.UiModePreviews
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun RadioButtonDemoScreen() = DemoScreen(rememberRadioButtonDemoState()) {
+    CustomizationBottomSheetScaffold(
+        bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+        bottomSheetContent = {
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_common_enabled_label),
+                checked = enabled,
+                onCheckedChange = { enabled = it },
+                enabled = enabledSwitchEnabled
+            )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_common_error_label),
+                checked = error,
+                onCheckedChange = { error = it },
+                enabled = errorSwitchEnabled
+            )
+        }
+    ) {
+        DetailScreenDescription(
+            modifier = Modifier.padding(all = OudsTheme.spaces.fixed.medium),
+            descriptionRes = Component.RadioButton.descriptionRes
+        )
+
+        LightDarkDemo {
+            RadioButtonDemo(state = this@DemoScreen)
+        }
+
+        RadioButtonDemoCodeSnippet(
+            state = this@DemoScreen,
+            modifier = Modifier
+                .padding(all = OudsTheme.spaces.fixed.medium)
+                .padding(top = OudsTheme.spaces.fixed.medium)
+        )
+    }
+}
+
+@Composable
+private fun RadioButtonDemo(state: RadioButtonDemoState) {
+    Row(
+        modifier = Modifier.selectableGroup(),
+        horizontalArrangement = Arrangement.Center
+    ) {
+        with(state) {
+            radioButtonDemoValues.forEach { value ->
+                OudsRadioButton(
+                    selected = value == selectedValue,
+                    onClick = { selectedValue = value },
+                    enabled = enabled,
+                    error = error
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun RadioButtonDemoCodeSnippet(state: RadioButtonDemoState, modifier: Modifier = Modifier) {
+    CodeSnippet(modifier = modifier) {
+        with(state) {
+            functionCall("OudsRadioButton") {
```

Comment:
For this kind of example, I don't know if we need to keep it simple like this although it might not be clear at first sight that this the code for the first button only, or if we should write the full code with the row and the two radio buttons.